Penteract/Assets/Scripts/PlayerController.h

```diff
@@ -158,9 +165,7 @@ class PlayerController : public Script
 	ComponentParticleSystem* onimaruCompParticle = nullptr;
 
 	//Audio
-	ComponentAudioSource* shootAudioSource = nullptr;
-	ComponentAudioSource* dashAudioSource = nullptr;
-	ComponentAudioSource* switchAudioSource = nullptr;
+	ComponentAudioSource* audios[static_cast<int>(AudioType::TOTAL)];
```

Comment:
Uninitialized. The values are undefined, not nullptr. This will pass an If statement and crash when use it. Initialize to  = { nullptr };